src/recorder.ts

```diff
@@ -168,7 +170,7 @@ export default async (
     const value = targetValue.result.value;
     const escapedValue = value.replace(/'/g, "\\'");
     const selector = await getSelector(client, targetId);
-    addLineToPuppeteerScript(`await type('${selector}', '${escapedValue}');`);
+    addLineToPuppeteerScript(`await type(${JSON.stringify(selector)}, ${JSON.stringify(escapedValue)});`);
```

Comment:
Should this be `JSON.stringify(value)`? AFAICT the escaping above was only done to try and make it fit in a single-quoted string literal

Reply:
Ah, yes that makes sense.